app/serializers/user_serializer.rb

```diff
@@ -4,6 +4,7 @@ class UserSerializer < ActiveModel::Serializer
   type "user"
   attributes :sso_guid, :created_at, :given_name, :family_name, :name
   has_many :tools, key: "favorite-tools", serializer: ResourceFavoritedSerializer
+  has_many :user_attributes
```

Comment:
we don't need to expose the raw `user_attributes` relationship through the serializer, this relationship is an internal implementation detail and nothing that we care about from API clients